src/test/java/Bellroy/BellroyTest.java

```diff
@@ -0,0 +1,34 @@
+package Bellroy;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import java.io.File;
+import java.io.IOException;
+public class BellroyTest {
+    private static final String testPath = "Bellroy_Test.txt";
+
+    @BeforeEach
+    public void setUp() {
+        File file = new File(testPath);
+        if (file.exists()) {
+            file.delete();
+        }
+    }
+
```

Comment:
The tests should also include JavaDoc comments too to clarify the purpose and scope of each test case.